src/extension/metadata/src/test/java/org/geoserver/metadata/EmptyConfigurationTest.java

```diff
@@ -0,0 +1,42 @@
+/* (c) 2023 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+
+package org.geoserver.metadata;
+
+import org.geoserver.data.test.MockData;
+import org.geoserver.metadata.web.MetadataTemplatesPage;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * Tests that metadata module works 'out of the box', without any configuration present yet.
+ *
+ * @author niels
+ */
+public class EmptyConfigurationTest extends AbstractWicketMetadataTest {
+
+    @BeforeClass
+    public static void init() throws Exception {
+        // this method shadows the init method from AbstractMetadataTest so that there
+        // is no metadata directory in the data directory.
+
+        // set data directory
+        MockData emptyDataDirectory = new MockData();
+        System.setProperty(
+                "GEOSERVER_DATA_DIR", emptyDataDirectory.getDataDirectoryRoot().toString());
```

Comment:
The JVMs in tests are re-used for the tests of the same module, but also across modules in a multi-module build. So please, make sure you don't leave sys var set up, clean them in a @AfterClass.

Reply:
Indeed I was having trouble with consecutive tests in maven  so I went for another approach